src/math/sat.rs

```diff
@@ -233,8 +232,8 @@ mod tests {
                 Point3::new( 1.0,  1.0, -1.0),
                 Point3::new( 1.0,  1.0,  1.0),
             ],
-            edges: unit_vectors.clone(),
-            face_normals: unit_vectors.clone(),
+            edges: ArrayVec::from_iter(unit_vectors.clone()),
+            face_normals: ArrayVec::from_iter(unit_vectors.clone()),
```

Comment:
See above

Reply:
`edges` and `face_normals` now differ in capacity (`ArrayVec<[T; 12]>` vs `ArrayVec<[T; 6]>`), and they both have more capacity than length (3, the number of elements in `unit_vectors`). So our options here are: 
* Have `unit_vectors` be an `ArrayVec<[T; 3]>` and use `from_iter()` for `edges` and  `face_normals`. That's pretty short and sweet, and has imo the best readability. Note we never need to mention an explicit capacity.
* Define two variables instead of one; `unit_vectors_edges` (an `ArrayVec<[T; 12]>`) and `unit_vectors_face_normals` (an `ArrayVec<[T; 6]>`), and `clone()` those. Maybe a little more efficient, but we shouldn't care about performance here obviously.
If there's a reason not to do it like this, I'm happy to reconsider, otherwise I prefer the present form. I can add comments as well of course.

Reply:
Ah, I've overseen this before.